DigitalLearningSolutions.Data.Tests/DataServices/RegistrationDataServiceTests.cs

```diff
@@ -51,6 +57,53 @@ public async Task Sets_all_fields_correctly_on_registration()
             user.Answer6.Should().Be(delegateRegistrationModel.Answer6);
         }
 
+        [Test]
+        public async Task RegisterAdmin_Sets_all_fields_correctly_on_centre_manager_admin_registration()

```

Comment:
This looks to be a duplicate of Sets_all_fields_correctly_on_centre_manager_admin_registration but now with the additional TCAgreed stuff. Do we want to get rid of that other test and move its contents to the new "...is_false" test?

Reply:
This was accidental duplication that I didn't tidy up. I was thinking of having the original test + a pair of true false tests that only checked TCAgreed, then decided not to do it, but forgot to delete the duplicate test.